docs/gds-supported-platform/resizing-persistent-volumes.md

```diff
@@ -0,0 +1,96 @@
+# Resizing persistent volumes
+
+The volumes you created in the beginning may not be big enough for
+your ongoing needs.
+
+As an example, we're going to double the space on the Concourse Worker
+persistent volumes - from 64Gi to 128Gi. Replace
+`gsp-concourse-worker` with the name of the resource you want to
+resize the volume for.
+
+1. Make a change to `charts/gsp-system/values.yml` with your desired
+   sizing, like [in PR 343](https://github.com/alphagov/gsp/pull/343):
+
+```yaml
+...
+persistence:
+  worker:
+    size: 128Gi
+```
+
+1. When merged, the PR will be released to sandbox. Things must pass
+   through sandbox via the full resize process to make sure we haven't
+   broken anything before going to the Verify cluster.
+
+1. Pods in StatefulSets each have a volume. For Kubernetes to be able
+   to resize the volumes, we need to detach them from the pods, by
+   destroying the stateful set. The `--cascade=false` flag is
+   important, as you don't want to delete the pods inside the stateful
+   set.
+
+   This step should be done as the Concourse release pipeline starts
+   to fail with an error message like:
+
+    > The StatefulSet “gsp-concourse-worker” is invalid: spec:
+    > Forbidden: updates to statefulset spec for fields other than
+    > ‘replicas’, ‘template’, and ‘updateStrategy’ are forbidden
+
+   To destroy the StatefulSet, run:
+
+    ```sh
+    $ gds sandbox kubectl -n gsp-system delete statefulset --cascade=false gsp-concourse-worker
+    ```
+
+   The `-deployer` pipeline will re-apply the statefulset. This will
+   intentionally roll all the associated pods.
+
```

Comment:
Content best practice is to put the action as close to the start of the numbered step as possible. The context is good, I would recommend rearranging the other content. For example:

"Pods in stateful sets each have a volume. For Kubernetes to be able to resize the volumes, we need to detach those volumes from the pods by destroying the stateful set. To destroy the stateful set, run:

    ```sh
    $ gds sandbox kubectl -n gsp-system delete statefulset --cascade=false gsp-concourse-worker
    ```
You should destroy the stateful set when the Concourse release pipeline starts to fail. You will see an error message similar to:

    > The StatefulSet “gsp-concourse-worker” is invalid: spec:
    > Forbidden: updates to statefulset spec for fields other than
    > ‘replicas’, ‘template’, and ‘updateStrategy’ are forbidden

The `--cascade=false` flag makes sure that you do not delete the pods inside the stateful set.

The `-deployer` pipeline re-applies the stateful set. This will intentionally roll all the associated pods.

Reply:
Also, will the user know what "roll all the associated pods" means?

Reply:
An SRE should definitely know what "roll the associated pods" means; we shouldn't be trying to turn this document into a "k8s fundamentals 101".